libvorbis.yaml

```diff
@@ -0,0 +1,60 @@
+package:
+  name: libvorbis
+  version: 1.3.7
+  epoch: 0
+  description: Vorbis codec library
+  copyright:
+    - license: BSD-3-Clause
+
+environment:
+  contents:
+    packages:
+      - busybox
+      - ca-certificates-bundle
+      - build-base
+      - libogg-dev
+
+pipeline:
+  - uses: fetch
+    with:
+      uri: https://downloads.xiph.org/releases/vorbis/libvorbis-${{package.version}}.tar.xz
+      expected-sha512: bbf5f11f623d959b3ba98d6b52806521b0b918b59b867f0658c153ea1013293364eead4987e9e71f8798021379310860a2e548fcf89d10caf23d45e41aba45cd
+
+  - runs: |
```

Comment:
Why not use normal `autoconf/configure`, `autoconf/make` and `autoconf/make-install` pipelines here?

Reply:
Thanks, done.